compiler/src/iree/compiler/Codegen/Dialect/GPU/TargetUtils/KnownTargets.cpp

```diff
@@ -146,7 +150,10 @@ const WgpDetails *getCDNA3WgpDetails() {
                                       {1024, 1024, 1024},
                                       1024,
                                       64 * 1024,
-                                      {0x7fffffff, 0x7fffffff, 0x7fffffff}};
+                                      {0x7fffffff, 0x7fffffff, 0x7fffffff},
+                                      /*maxLoadInstructionBits=*/128,
+                                      /*workgroupSimds=*/4,
```

Comment:
You can safely use 128 and 4 for all amd and nvidia chips defined here.

Reply:
Ah, good to know... but since we have other targets in this file, I don't immediately see how that allows us to not make this optional. Filed https://github.com/iree-org/iree/issues/18849.